### PR TITLE
Update styling to be closer to material design

### PR DESCRIPTION
**Pull request by chuckhays**
Updates styling to be close to the material design spec.

https://github.com/GoogleChrome/webrtc/issues/410

@samdutton @juberti

### CODE REVIEW

File: samples/web/content/apprtc/css/main.css [hunk 1]
```diff
@@ -168,44 +168,80 @@ label {
...
+input[type="text"]:focus {
+  border-bottom: solid 2px #4285F4;
+  outline: none;
+}
+
+input[type="text"].invalid {
+  border-bottom: solid 2px #F44336;
+}
+
+label.errorLabel {
```
Comment (juberti) [inline samples/web/content/apprtc/css/main.css]: style for class names is error-label

**Comment by juberti** [comment]
HTML/CSS LG. I would horizontally center this, and maybe add some vertical padding. It looks a bit lonely up in the upper LH corner.

**Comment by chuckhays** [comment]
Bit hard to see in the screenshot, but content is centered, moved down a bit, and the input is made narrower. It looked awkward to have it so wide once the content was centered. The disadvantage is that the hint text wraps.

screen shot 2015-01-23 at 2 12 27 pm

**Comment by juberti** [comment]
lgtm

**Pull request merged**